js/packages/phoenix-client/src/lib/api/v1.d.ts

```diff
@@ -0,0 +1,1134 @@
+/**
+ * This file was auto-generated by openapi-typescript.
+ * Do not make direct changes to the file.
+ */
+
+export interface paths {
+  "/v1/datasets": {
+    parameters: {
+      query?: never;
+      header?: never;
+      path?: never;
+      cookie?: never;
+    };
+    /** List datasets */
+    get: operations["listDatasets"];
+    put?: never;
+    post?: never;
+    delete?: never;
+    options?: never;
+    head?: never;
+    patch?: never;
+    trace?: never;
+  };
+  "/v1/datasets/{id}": {
+    parameters: {
+      query?: never;
```

Comment:
@cephalization this is weird - is it something we can configure? it should be at least any here

Reply:
Something is definitely messed up. The last openapi codegen tool I used strongly typed the query parameters of each endpoint and I'd expect the same here

Reply:
<img width="838" alt="image" src="https://github.com/user-attachments/assets/2bb40b84-e393-4b2e-b78e-31aae3c3aac2">

Looks like later on in the generated types it overrides these `never` types with the appropriate types